default-views/account/invalid-username.hbs

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Invalid username</title>
+  <link rel="stylesheet" href="/common/css/bootstrap.min.css">
+  <script></script>
+</head>
+<body>
+<div class="container">
+  <h4>Invalid username</h4>
```

Comment:
Why not an `h1`? Semantic Web and so on.

Reply:
Because of legacy; the other views use h4 (I reckon they did it like this initially because they thought h1 was to big; We (mostly) use bootstrap (3.3.4) for the presentation).

Reply:
@megoth hehe, that's exactly why, h1 was too big on bootstrap :)

Reply:
https://getbootstrap.com/docs/3.3/customize/#typography *cough